packages/env/lib/config.js

```diff
@@ -78,6 +79,18 @@ module.exports = {
 			}
 		}
 
+		try {
+			overrideConfig = JSON.parse(
+				await fs.readFile(
+					configPath.replace(
+						/\.wp-env\.json$/,
+						'.wp-env.override.json'
+					),
+					'utf8'
+				)
+			);
+		} catch ( err ) {}
```

Comment:
Let's handle the error similarly to the lines above by throwing `ValidationError` when the file can't be read or when the JSON is invalid. It's useful for troubleshooting. I for one am constantly adding trailing commas to my JSON file 🙂

Reply:
Done